pkg/workflow/controller/handlers/workflowrun/handler.go

```diff
@@ -107,7 +107,7 @@ func (h *Handler) ObjectUpdated(obj interface{}) {
 	// the GC queue.
 	h.GCProcessor.Add(originWfr)
 
-	// If the WorkflowRun has already been terminated(Completed, Error, Cancel), send notifications if necessary,
+	// If the WorkflowRun has already been terminated(Completed, Error), send notifications if necessary,
```

Comment:
I think we also need to send notify even if it is canceled, @supereagle  WDYT?

Reply:
In my next PR, will send notification for all wfrs.